build/build.sh

```diff
@@ -27,7 +27,7 @@ readonly dicom_store_name="$(openssl rand -hex 12)"
 readonly mount_folder="dicom"
 mkdir "${mount_folder}"
 # Create unique DICOM Store
-gcloud alpha healthcare dicom-stores create "${dicom_store_name}" \
+gcloud beta healthcare dicom-stores create "${dicom_store_name}" \
   --location="${LOCATION}" \
   --dataset="${DATASET}" \
   --quiet
```

Comment:
Let's fail early here if gcloud fails, so that we don't get a bunch of other irrelevant error messages.

Reply:
Done in [the](https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/pull/42) separate PR.